BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -187,4 +194,71 @@ extension SideMenuFilterViewController {
 
     }
 
+    
+    func setUserPreferences() {
+        
+        defaultPreferences.set(distanceFilter, forKey: "distanceFilter")
+        defaultPreferences.set(isRestaurant, forKey: "isRestaurant")
+        defaultPreferences.set(isOpenNow, forKey: "isOpenNow")
+        defaultPreferences.set(isStore, forKey: "isStore")
+        
+    }
+    
+    func loadDefaults() {
+        
+        if defaultPreferences.double(forKey: "distanceFilter") > 0 {
```

Comment:
might be better to start with a `guard` and `return` if it's false. That way the rest of this function doesn't happen on an indented block.